oddsgym/envs/tennisdata.py

```diff
@@ -0,0 +1,107 @@
+import itertools
+import os
+import numpy
+import pandas
+import gym
+from .tennis import TennisOddsEnv
+from ..utils.constants.tennis import CSV_CACHE_PATH, CSV_URL, TOURNAMENTS, SITES, YEARS
+
+
+class TennisDataMixin(object):
```

Comment:
should be in the same class or as a separated one?

Reply:
I want to rethink the whole design here; though about doing it in this branch, but it looked like a big detour.
We can redesign how we build those classes in a separate branch (like the metaprogramming one).